### ISSUE TITLE
No cursor in sequence name field

### ISSUE THREAD

**Issue by simonvanderveldt** [ahlstromcj/sequencer64#52]
When editing a sequence the field in the top left, which holds the sequence's name, doesn't show a cursor when editing the name. This makes editing the name a bit trickier than it should be ;)

**Comment by ahlstromcj** [comment]
Yeah, this is an issue in a number of fields all over the application.  Some fields show the cursor, some don't and I have no idea why.  Let me know if you think of something!

**Comment by AndrewPy** [comment]
I think it has something to do with the fact that some windows are modal (e.g. the options window - there is a visible cursor), and some windows are non modal (for example the event editor or the piano roll editor - in this case cursor and focus are visible in the main window only).
